File: ListadoPersonasYDepartamentos/Domain/Entities/ResultadoJuego.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Domain.Entities
{
    public class ResultadoJuego
    {
        public int Aciertos { get; set; }
        public int Total { get; set; }
        public bool Gano { get; set; }
    }
}

File: tema8/Select-persona-Azure/app/domain/entities/ResultadoJuego.ts
export class ResultadoJuego {
  aciertos: number;
  total: number;
  gano: boolean;

  constructor(aciertos: number, total: number, gano: boolean) {
    this.aciertos = aciertos;
    this.total = total;
    this.gano = gano;
  }
}